src/param.rs

```diff
@@ -1110,9 +1110,9 @@ impl AudioParamProcessor {
 
         if infos.is_a_rate {
             let start_index = self.buffer.len();
-            // we need to `ceil()` because if `end_time` is between two samples
+            // we need to `round()` because if `end_time` is between two samples
```

Comment:
no sure these comments still hold then... which I consider is a bit strange actually